samples/node-headless-ssr-experience-edge/config.js

```diff
@@ -32,6 +32,11 @@ const config = {
    * Port which will be used when start sample
    */
   port: process.env.PORT || 3000,
+  /*
+  * The default language to use in case the context language cannot be determined (this happens
+  * on initial page load, if the language is not specified in the URL)
+  */
+  defaultLanguage: process.env.DEFAULT_LANGUAGE || serverBundle.defaultLanguage
```

Comment:
`DEFAULT_LANGUAGE` should be added to [readme](https://github.com/Sitecore/jss/blob/dev/samples/node-headless-ssr-experience-edge/README.md#environment-variables).

Reply:
Done